eclair-core/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -198,6 +198,26 @@ trait Service extends Logging {
                           completeRpcFuture(req.id, (paymentHandler ? ReceivePayment(Some(MilliSatoshi(amountMsat.toLong)), description)).mapTo[PaymentRequest].map(PaymentRequest.write))
                         case _ => reject(UnknownParamsRejection(req.id, "[description] or [amount, description]"))
                       }
+                      
+                      case "checkinvoice" => req.params match {
+                        case JString(paymentRequest) :: Nil  => Try(PaymentRequest.read(paymentRequest)) match {
+                          case Success(pr) => {
```

Comment:
nitpick: remove unnecessary braces around `completeRpc`

Reply:
Arh - using the router is much simpler! Updated.
Use-case for both these is another app using the API - in checkinvoice case is to check amount is what we are expecting - before paying with send. So zeros fine.